eslintrc.json

```diff
@@ -27,6 +28,8 @@
     "react/no-unknown-property": "error",
     "react/no-unused-prop-types": "error",
     "react/prop-types": "error",
-    "react/react-in-jsx-scope": "error"
+    "react/react-in-jsx-scope": "error",
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "error"
```

Comment:
I _think_ that this could result in a lot of errors for many project, potentially it would be better to first release this as `warning` and then switch to `error` in the major version after that.

(that is how `no-var` was rolled out: https://github.com/standard/standard/pull/1586)

Reply:
Sounds good ! For some reason I thought the policy for standard eslint rules was no warnings, only errors.

Reply:
Done @LinusU